staging/src/k8s.io/apiextensions-apiserver/pkg/features/kube_features.go

```diff
@@ -29,6 +29,7 @@ const (
 
 	// owner: @sttts, @nikhita
 	// alpha: v1.8
+	// beta: v1.9
```

Comment:
not sure of the expected/desired syntax here; let me know what's appropriate.

Reply:
seems right as per:

https://github.com/kubernetes/kubernetes/blob/7914aed80a7af9b9cd27b6e7ffa3d8721886c2ed/staging/src/k8s.io/apiserver/pkg/features/kube_features.go#L59-L65

Reply:
👍